tests/components/test_graphite.py

```diff
@@ -43,20 +40,22 @@ def test_full_config(self, mock_gf):
                 'prefix': 'me',
             }
         }
+
         self.assertTrue(graphite.setup(self.hass, config))
         mock_gf.assert_called_once_with(self.hass, 'foo', 123, 'me')
 
-    @mock.patch('homeassistant.components.graphite.GraphiteFeeder')
-    def test_config_bad_port(self, mock_gf):
```

Comment:
Because I removed the type check.

```python
    try:
        port = int(graphite_config.get('port', 2003))
    except ValueError:
        _LOGGER.error('Invalid port specified')
        return False
```
The port check performed by `voluptuous` should make sure that port is an integer.